src/item/TurtleHelmet.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ *
+ *  ____            _        _   __  __ _                  __  __ ____
+ * |  _ \ ___   ___| | _____| |_|  \/  (_)_ __   ___      |  \/  |  _ \
+ * | |_) / _ \ / __| |/ / _ \ __| |\/| | | '_ \ / _ \_____| |\/| | |_) |
+ * |  __/ (_) | (__|   <  __/ |_| |  | | | | | |  __/_____| |  | |  __/
+ * |_|   \___/ \___|_|\_\___|\__|_|  |_|_|_| |_|\___|     |_|  |_|_|
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * @author PocketMine Team
+ * @link http://www.pocketmine.net/
+ *
+ *
+ */
+
+declare(strict_types=1);
+
+namespace pocketmine\item;
+
+use pocketmine\entity\effect\EffectInstance;
+use pocketmine\entity\effect\VanillaEffects;
+
+class TurtleHelmet extends Armor{
+
+	/**
+	 * @return EffectInstance[]
+	 * @phpstan-return list<EffectInstance>
+	 */
+	public function getEffects() : array{
```

Comment:
This function feels pointless. You are already hardcoding the activation logic for this, so I don't see why add an extra class.

Reply:
I was thinking about an alternative, would be to add new variables to ArmorTypeInfo, would be the armor effects and maybe a cooldown to apply them.

Reply:
The suggested changes have been applied